src/movescount/movescountjson.cpp

```diff
@@ -839,23 +839,27 @@ bool MovesCountJSON::writePeriodicSample(ambit_log_sample_t *sample, QVariantMap
 
         switch(value->type) {
         case ambit_log_sample_periodic_type_latitude:
-            output.insert("Latitude", (double)value->u.latitude/10000000);
+            if (value->u.latitude != 0xffffffff && value->u.latitude <= 90 && value->u.latitude >= -90){
```

Comment:
CLion warns that the comparison to 0xf.. is redundant, probably because the variable is a signed int, are you sure this is needed?

Reply:
On 2nd thought, maybe the check is redundant now because the condition on "> and <" already excludes this value now as well.